app/index.coffee

```diff
@@ -46,14 +47,14 @@ User.bind 'sign-in', ->
     ExperimentServer.resetExperimentalFlags()
     Geordi.logEvent 'logout'
 
-Api.init
-  host: if !!location.href.match /demo|beta/
-    'https://dev.zooniverse.org'
-  else if +location.port < 1024
-    'https://api.zooniverse.org'
-  else
-    'https://dev.zooniverse.org'
-    #"#{location.protocol}//#{location.hostname}:3000"
+[host, proxyPath] = if location.origin is 'http://preview.zooniverse.org'
```

Comment:
this is a merge of the already committed change here: https://github.com/zooniverse/Serengeti/commit/7bc0359471ed68645099fab5960fd4816b54fe10
not quite sure why it's showing as a diff.